File: Algoritmos_e_Logica_de_Programacao/Exercicios_logica_em_C/04-vetores/Soma_vetor/Soma_vetor.c
#include <stdio.h>

int main()
{

    int casos;
    printf("Quantos numeros voce vai digitar? ");
    scanf("%d", &casos);

    printf("\n");

    double numeros[casos];
    for (int i = 0; i < casos; i++) {
        printf("Digite um numero: ");
        scanf("%lf", &numeros[i]);
    }

    double soma = 0;
    printf("\nValores = ");
    for (int i = 0; i < casos; i++) {
        printf("%.1lf ", numeros[i]);
        soma += numeros[i];
    }

    double media = soma / casos;
    printf("\nsoma = %.2lf\n", soma);
    printf("Media = %.2lf", media);

    return 0;
}

File: Curso_logica_de_programacao/Exercicios_logica_em_Java/Soma_vetor/src/Soma_vetor.java
import java.util.Scanner;
import java.util.Locale;

public class Soma_vetor {
    public static void main(String[] args) throws Exception {

        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);

        System.out.print("Quantos numeros voce vai digitar? ");
        int n = sc.nextInt();

        double[] vet = new double[n];
        for (int i = 0; i < n; i++) {
            System.out.print("Digite um numero: ");
            vet[i] = sc.nextDouble();
        }

        System.out.print("Valores = ");
        for (int i = 0; i < n; i++) {
            System.out.print(" " + vet[i]);
        }
        System.out.println();

        double soma = 0;
        for (int i = 0; i < n; i++) {
            soma += vet[i];
        }
        System.out.println("Soma = " + String.format("%.2f", soma));

        double media = soma / n;
        System.out.println("Media = " + String.format("%.2f", media));

        sc.close();
    }
}
